packages/jellyfish-api-core/src/category/rawtx.ts

```diff
@@ -335,3 +345,55 @@ export interface RawTransaction {
    */
   blocktime: number
 }
+
+export interface DecodeScriptResult {
+  asm: string
+  /**
+   * Script public key
+   */
```

Comment:
```suggestion
  /**
   * Script public key
   */
    asm: string
```

Ehm, comment should appear before field.